### ISSUE TITLE
Memory Leak

### ISSUE THREAD

**Issue by unclearimage86** [godotengine/godot#33296]
Tried website binaries, steam version, and MINGW & MSVS compiled on windows 10.

Running an app with a GDscript attached increases the run time by about one second. Until eventually after 6-7 runs it takes up to 45 seconds for the app to even run.

Steps to repeat:
Created a node
Attach a Script
Run, then close app by clicking close button ('x') on application window and run immediately run again.
Time for the application to boot increases each time until it reaches 45+ seconds

**Comment by Calinou** [comment]
Are you using a NVIDIA graphics card? This sounds similar to #20566.

**Comment by unclearimage86** [comment]
> 
> 
> Are you using a NVIDIA graphics card? This sounds similar to #20566.
> 
> What's the RAM usage of the exported project as measured by the Task Manager?

I have an NVIDIA GeForce GTX 1060 3gb with driver 441.08 (10/29/2019) 

I checked out the issue #20566 but they said it was a USB problem. I've removed all USB drivers that aren't being used. Whatever the case, there's no solution I saw there.

Godot is using up to 175 mb of Memory while trying to run. 161 mb while running, and the app that runs uses 60mb. Neither uses more than 1% of my cpu.
